common-ktx/src/main/java/co/nimblehq/common/extensions/date/DateRange.kt

```diff
@@ -0,0 +1,13 @@
+package co.nimblehq.common.extensions.date
+
+import java.util.*
+
+/**
+ * A range of values of type [Date].
+ */
+class DateRange(
+    override val start: Date,
+    override val endInclusive: Date
+) : ClosedRange<Date> {
+    override fun contains(value: Date) = start < value && value < endInclusive
```

Comment:
Please add a break-line here. 🙏🏻 
```suggestion

    override fun contains(value: Date) = start < value && value < endInclusive
```